video/video.ino

```diff
@@ -253,6 +250,24 @@ int	readWord_t() {
 	return -1;
 }
 
+// Read an unsigned 24-bit value from the serial port, with a timeout
+// Returns:
+// - Value (0 to 16777215) if 3 bytes read, otherwise -1
+//
+int	read24_t() {

```

Comment:
used for uploading samples, as a 16-bit length value only allows for a 4s sample

our available memory is only 4MB so a 24-bit value is more than enough